src/main/java/com/softserve/teamproject/config/WebSecurityConfig.java

```diff
@@ -9,49 +11,55 @@
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
-import org.springframework.security.core.AuthenticationException;
+import org.springframework.security.web.authentication.SimpleUrlAuthenticationFailureHandler;
 
-/**
- * The class provides basic security configuration.
- */
 @Configuration
 @EnableWebSecurity
 public class WebSecurityConfig extends WebSecurityConfigurerAdapter {
 
-  @Autowired
-  private UserDetailsServiceImpl userDetailsService;
+    @Autowired
+    private UserDetailsServiceImpl userDetailsService;
+    @Autowired
+    RestAuthenticationEntryPoint restAuthenticationEntryPoint;
+    @Autowired
+    CustomSavedRequestAwareAuthenticationSuccessHandler successHandler;
 
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
+        //@formatter:off
+        http
+              .exceptionHandling().authenticationEntryPoint(restAuthenticationEntryPoint).and()
+              .authorizeRequests()
+                .antMatchers("/**").authenticated().and()
+              .formLogin()
+                .loginPage("/login").permitAll()
+                .successHandler(successHandler)
+                .usernameParameter("username")
+                .passwordParameter("password")
+                .failureHandler(new SimpleUrlAuthenticationFailureHandler()).and()
+              .logout()
+                .logoutUrl("/logout")
+                .logoutSuccessUrl("/").and()
+              .rememberMe().key("token").tokenValiditySeconds(3600);
```

Comment:
Didn't you agree with @andrewkononenko that call indentation should be 4 spaces?